{"ID": "c81dd96f-9b2a-4340-bfb9-6dfc6868fce7", "effective_cve": "CVE-2021-25642", "image": {"exact": "docker.io/bitnami/spark@sha256:267d5a6345636710b4b57b7fe981c9760203e7e092c705416310ea30a9806d74"}, "label": "FP", "package": {"name": "hadoop-auth", "version": "3.3.1"}, "timestamp": "2023-05-10T14:20:17.706889", "tool": "grype@v0.61.1", "user": "westonsteimel", "vulnerability_id": "CVE-2021-25642"}